### DISCUSSION TITLE
Issues building examples on MacOS

### DISCUSSION THREAD

**Discussion by nbentall** [iPlug2/iPlug2#958] [Q&A]
Trying to build example effect on macOS in Xcode, (m1 processor, Ventura 13.3)

getting error with plist formatting it seems?

any suggestions how to fix this? Is this related to plist library code in python being fussy about syntax of plist file?

Thanks. 

PhaseScriptExecution Run\ Script\ -\ prepare_resources-mac.py /Users/nb/Library/Developer/Xcode/DerivedData/IPlugEffect-cvhypwkvgljicygmvvtmtcfmwbjx/Build/Intermediates.noindex/IPlugEffect-macOS.build/Debug/AU.build/Script-4FBCC1A21FD59C7300EFE550.sh (in target 'AU' from project 'IPlugEffect-macOS')
    cd /Users/nb/iplug/iPlugR2/iPlug2/Examples/IPlugEffect/projects
    /bin/sh -c /Users/nb/Library/Developer/Xcode/DerivedData/IPlugEffect-cvhypwkvgljicygmvvtmtcfmwbjx/Build/Intermediates.noindex/IPlugEffect-macOS.build/Debug/AU.build/Script-4FBCC1A21FD59C7300EFE550.sh

Copying resources ...
copying Roboto-Regular.ttf to /Users/nb/Library/Audio/Plug-Ins/Components/IPlugEffect.component/Contents/Resources
Processing Info.plist files...
Traceback (most recent call last):
  File "/Users/nb/iplug/iPlugR2/iPlug2/Examples/IPlugEffect/projects/../scripts/prepare_resources-mac.py", line 215, in <module>
    main()
  File "/Users/nb/iplug/iPlugR2/iPlug2/Examples/IPlugEffect/projects/../scripts/prepare_resources-mac.py", line 141, in main
    auv3 = plistlib.load(fp)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/plistlib.py", line 869, in load
    raise InvalidFileException()
plistlib.InvalidFileException: Invalid file
Command PhaseScriptExecution failed with a nonzero exit code

**Comment by AlexHarker** [comment]
It looks like the process might have failed once, leaving you without the plist file on disk. Subsequent calls with then fail.

I'd advise that you check to see if the repo has changes on the plist files and if so discard them and try again.
